scripts/setup-gcloud-cluster.sh

```diff
@@ -197,7 +199,7 @@ for instance_idx in "${!instance_names_arr[@]}" ; do
   wait_until_node_is_ready "${node_ip}"
 
   if [[ "${reset_metadata}" = "true" ]]; then
-    gcloud_remove_instance_metadata "${instance_name}" "hopr-peer-id,hopr-wallet-addr,hopr-staking-addr"
+    gcloud_remove_instance_metadata "${instance_name}" "HOPRD_PEER_ID,HOPRD_WALLET_ADDR,HOPRD_STAKING_ADDR"
```

Comment:
What happens with the existing instances? If metadata gets renamed, the deployment won't migrate metadata to the new name.

Reply:
When this PR is merged, I need to delete manually the existing cluster to take effect the new instance-template name and the metadata, I already have in mind that. That will happen as well for applying(merging) to master branch.